wp_links_artists.py

```diff
@@ -262,7 +262,7 @@
                 continue
 
         wp_url = 'http://%s.wikipedia.org/wiki/%s' % (wp_lang, quote_page_title(page_title),)
```

Comment:
This is only used in the edit note. Perhaps it should still be changed to https, too?